freeglut/freeglut/src/mswin/fg_main_mswin.c

```diff
@@ -1544,13 +1544,14 @@ LRESULT CALLBACK fgPlatformWindowProc( HWND hWnd, UINT uMsg, WPARAM wParam, LPAR
     }
 #endif
 
-#ifdef WM_INPUT
+#ifdef WM_INPUT	
+	case WM_INPUT_DEVICE_CHANGE:	/* Added by Shane Saxon. Dec 2020 */
+		fgSpaceballDeviceChangeWinEvent( hWnd, wParam, lParam );
```

Comment:
Match the indentation of the surrounding block. The rest of the code in this part uses spaces, not tabs.